web/profiles/custom/yalesites_profile/config/sync/filter.format.basic_html.yml

```diff
@@ -16,7 +16,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <br> <span> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title> <table> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <sup> <sub>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol type start> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <br> <span> <table> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <sup> <sub> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title target !href>'
```

Comment:
@marcb4k Andy idea what `!href` is?

Reply:
Hum, I didn't add/change that from what it was but that is odd. I can double-check that one and fix/remove it. It obviously should allow href!